lib/delayed_resque/version.rb

```diff
@@ -1,3 +1,3 @@
 module DelayedResque
-  VERSION = "1.0.9"
+  VERSION = "1.0.10"
```

Comment:
Handling metadata could be considered a bigger bump than a patch level. Consider `1.1.0`?